lms/djangoapps/bulk_email/tasks.py

```diff
@@ -497,6 +497,7 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
         # use the email from address in the CourseEmail, if it is present, otherwise compute it
         from_addr = course_email.from_addr if course_email.from_addr else \
             _get_source_address(course_email.course_id, course_title, course_language)
+    from_addr = "{} course team <{}>".format(course_title, from_addr)
```

Comment:
Kindly see the implementation of `_get_source_address`, It seems me it is making from address something like that pattern already 
`"COURSE_TITLE" Course Staff <course_name-no-reply@courseupdates.edx.org>`
so with your new change, title will be added again if flow goes through the `_get_source_address`